packages/arb-token-bridge-ui/src/hooks/__tests__/useTransferDuration.test.ts

```diff
@@ -0,0 +1,439 @@
+import { RenderHookResult, act, renderHook } from '@testing-library/react'
+import { MergedTransaction } from '../../state/app/state'
+import { AssetType } from '../../hooks/arbTokenBridge.types'
+import dayjs from 'dayjs'
+import {
+  DEPOSIT_TIME_MINUTES_MAINNET,
+  DEPOSIT_TIME_MINUTES_ORBIT_MAINNET,
+  DEPOSIT_TIME_MINUTES_ORBIT_TESTNET,
+  DEPOSIT_TIME_MINUTES_TESTNET,
+  TRANSFER_TIME_MINUTES_CCTP_MAINNET,
+  TRANSFER_TIME_MINUTES_CCTP_TESTNET,
+  useTransferDuration
+} from '../../hooks/useTransferDuration'
+
+const DAY_IN_MINUTES = 24 * 60
+const HOUR_IN_MINUTES = 60
+
+function mockTransactionObject({
+  minutesSinceStart,
+  isDeposit,
+  isCctp,
+  parentChainId,
+  childChainId
+}: {
+  minutesSinceStart: number
+  isDeposit: boolean
+  isCctp: boolean
+  parentChainId: number
+  childChainId: number
+}): MergedTransaction {
+  return {
+    sender: '',
+    destination: '',
+    direction: 'deposit',
```

Comment:
Could be withdrawal but this field doesn't matter for time estimation. We use `isWithdrawal` only.